src/cfnlint/core.py

```diff
@@ -176,7 +177,11 @@ def append_parser(parser, defaults):
         version='%(prog)s {version}'.format(version=__version__)
     )
     parser.add_argument(
-        '--update-specs', help='Update the CloudFormation Specs',
+        '--update-specs', help=argparse.SUPPRESS,
```

Comment:
Same here. Why suppress? Give the folks the option to update if they add local rules or want the latest spec.

Reply:
That still works, just "hidden it", put it back to what it was?